frontend/Dockerfile

```
@@ -0,0 +1,26 @@
+# Use an official Node.js runtime as a parent image
+FROM node:14
```

Let's not use an unsupported node version.
I would recommend Node 22 with a slim configuration, so e.g.:

```suggestion
FROM node:22-bookworm-slim
```

---

See the list of Docker tags for node here: https://hub.docker.com/_/node